packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -1360,7 +1360,7 @@ export function emitsOpenCloseEvents(
     element.setProperty(toggleProp, true);
     await page.waitForChanges();
 
-    const container = await page.find(`calcite-dropdown-group >>> ${containerSelector}`);
+    const container = await page.find(containerSelector);
```

Comment:
I think we can focus solely on the events and assume the element will be visible (or not visible) to avoid checking visibility. This should make the helper simpler to use.